README.md

```diff
@@ -50,3 +82,15 @@
 
 ## Advance ML-on-Code awareness
 
+- Is it supposed to be MLonCode instead of ML-on-Code? - Vadim
+
+### ML team activities
+
+- [ ] Speak on two conferences
+    - This is an "ambitious goal" because ML team would like to focus on the intelligent code review. The one conference being ML4P in mid-July.
+- [ ] Write 5 ML blog posts
```

Comment:
ML Team doesn't have time in this Q3 for write blog post, unless are strictly related with the analyzer and we are on track.

Reply:
Sorry @campoy @vcoisne I did my best here. I will still write a few in my spare time though.